### ISSUE TITLE
Update helidon-mp example

### ISSUE THREAD

**Issue by hg-ms** [microstream-one/microstream#425]
Our Helidon-MP example currently uses Helidon 2.3.2.
When updating to Helidon version 2.5.2 the example will build throw an exception at startup:

```
Exception in thread "main" java.util.ServiceConfigurationError: org.eclipse.microprofile.config.spi.Converter: Provider
one.microstream.integrations.cdi.types.config.StorageManagerConverter could not be instantiated
```

I was able to fix that by adding the jul-to-slf4j depenency as a quick  fix.
``` 
<dependency>
    <groupId>org.slf4j</groupId>
    <artifactId>jul-to-slf4j</artifactId>
</dependency>
``` 

We should update the heldion version used by our example and check the required dependencies

**Comment by rdebusscher** [comment]
Issue is due to Helidon 2.4+ containing older versions of MicroStream (due to Helidon integration and using the Helidon POM import.

See https://github.com/microstream-one/microstream/pull/444

**Comment by rdebusscher** [comment]
Fixed by #444
